docs/user/openstack/README.md

```diff
@@ -646,14 +648,26 @@ set-cookie: 1e2670d92730b515ce3a1bb65da45062=9b714eb87e93cf34853e87a92d6894be; p
 cache-control: private
 ```
 
-### Refreshing a CA Certificate
+### Reconfiguring cloud provider
 
-If you ran the installer with a [custom CA certificate](#self-signed-openstack-ca-certificates), then this certificate can be changed while the cluster is running. To change your certificate, edit the value of the `ca-cert.pem` key in the `cloud-provider-config` configmap with a valid PEM certificate.
+If you need to update the OpenStack cloud provider configuration you can edit the ConfigMap containing it:
 
 ```sh
 oc edit configmap -n openshift-config cloud-provider-config
 ```
 
+There are several things you can change:
+
+#### Modifying cloud provider options
+
+If you need to modify the direct cloud provider options, then edit the `config` key in the ConfigMap. A brief list of possible options is shown in [Cloud Provider configuration](./customization.md#cloud-provider-configuration) section.
+
+Please note that it can take a while to reconfigure the cluster depending on the size of it. The reconfiguration is completed once no node is getting `SchedulingDisabled` taint anymore.
```

Comment:
Should we move the note outside of the subsection,  on line 658, since it applies to bot modifying cloud provider options and refreshing the CA certificate?

Reply:
Your call, I only noticed that it involves the same file, but I'm okay with reverting to the previous version.

Reply:
What was the previous version? I'm asking to move line 665 a few lines above so that the note applies to both `modifying cloud provider options` and `refreshing the CA certificate` subsections.

Reply:
Ah, okay, I misunderstood!